README.md

```diff
@@ -0,0 +1,57 @@
+# Coveo Cloud services' official `.js` client
+
+The main goal of this package is to provide an easy to configure and straightforward way of querying any API fronted by the Coveo Cloud's services using javascript.
```

Comment:
The main goal of this package is to provide an easy to configure and straightforward way of querying Coveo Cloud APIs using JavaScript.